### DISCUSSION TITLE
What are all possible configuration entries for a vm / template?

### DISCUSSION THREAD

**Discussion by madalinignisca** [lima-vm/lima#2434] [Q&A] [answered]
Hi. I learned many by getting examples from network.yaml examples and others for templates from docs and occasionally from issues in the repo.

Like for example that I can disable containerd, which previously I was creating vms with --plain argument all the time to skip it, but made me think I am skipping possible other stuff I would like enabled.

Where is a list of all configuration supported by a template, to learn what extra configuration we'd need to ensure we provision the vms as expected?

**Comment by afbjorklund** [comment, marked as answer]
The documentation is in the [default.yaml](https://github.com/lima-vm/lima/blob/master/examples/default.yaml). Currently the only required field is the `images`.

**Reply by madalinignisca** [reply]
Awesome. Thank you. default.yaml was the only one I did not look into, as I must work only with debian and ubuntu and expect close to the official cloud things, nothing added to them.

**Reply by afbjorklund** [reply]
The default.yaml is currently the same as ubuntu.yaml, but with extra comments added.

**Comment by jamagin** [comment]
For anyone like me who comes along and can't find this, it's moved to [templates/default.yaml](https://github.com/lima-vm/lima/blob/master/templates/default.yaml)
